tests/acceptance/10_files/02_maintain/absolute_link_and_no_move_obstructions.cf

```diff
@@ -44,8 +44,8 @@ bundle agent test
 
   files:
       "$(G.testfile)"
-      classes => init_if_failed("test_ok"),
-      link_from => test_link;
+          classes => init_if_failed("test_ok"),
+          link_from => test_link;
```

Comment:
The official CFEngine policy standard is two spaces per level, so that will be:

```
{
  files:
    class::
      "promiser"
        attribute => "value";
}
```

If class is missing you still behave as if it were there.

I know, this is violated a lot, but if you're fixing it anyway...